packages/perseus/src/index.ts

```diff
@@ -36,7 +36,23 @@ export {default as TableWidget} from "./widgets/table";
 export {default as PlotterWidget} from "./widgets/plotter";
 export {default as GrapherWidget} from "./widgets/grapher";
 
-export * as GrapherUtil from "./widgets/grapher/util";
+import {
+    allTypes,
+    DEFAULT_GRAPHER_PROPS,
+    chooseType,
+    defaultPlotProps,
+    getEquationString,
+    typeToButton,
+} from "./widgets/grapher/util";
+
+export const GrapherUtil = {
```

Comment:
Love it.  Could you just add a comment on this export explaining why we're doing this (namely, the exported functions make more sense as they're scoped inside the `GrapherUtil` object).